axis_backend/apps/audit/services/entity_change_service.py

```
@@ -0,0 +1,101 @@
+
+    @transaction.atomic
+    def record_change(
+        self,
+        entity_type: str,
+        entity_id: str,
+        change_type: str,
+        changed_by: Optional[str] = None,
+        change_reason: Optional[str] = None,
+        old_data: Optional[dict] = None,
+        new_data: Optional[dict] = None,
+        **kwargs
```

Ruff (0.14.5)

Missing type annotation for `**kwargs`

(ANN003)